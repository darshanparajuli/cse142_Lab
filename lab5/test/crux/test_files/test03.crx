// test return type matches function

func myInt(a:int, b:float) : int {
  return a;
}

func myIntError(a:float) : int {
  return a;
}

func main() : void {
  ::myInt(7);
  ::myIntError(7.0);
}
