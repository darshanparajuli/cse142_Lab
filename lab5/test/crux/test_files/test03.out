Error type-checking file.
TypeError(8,3)[Function myIntError returns int not float.]

